src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -344,24 +355,33 @@ def __new_assessment_summary_json(self, assessment_packages_json, status, code):
         # discern started by - either pure auto-assessment or assessment data being included with configure patching with assessmentMode set to AutomaticByPlatform
         started_by = Constants.PatchAssessmentSummaryStartedBy.PLATFORM if (self.execution_config.exec_auto_assess_only or self.execution_config.include_assessment_with_configure_patching) else Constants.PatchAssessmentSummaryStartedBy.USER
 
-        # Compose sub-status message
-        substatus_message = {
-            "assessmentActivityId": str(self.execution_config.activity_id),
-            "rebootPending": self.is_reboot_pending,
-            "criticalAndSecurityPatchCount": critsec_patch_count,
-            "otherPatchCount": other_patch_count,
-            "patches": assessment_packages_json,
-            "startTime": str(self.execution_config.start_time),
-            "lastModifiedTime": str(self.env_layer.datetime.timestamp()),
-            "startedBy": str(started_by),
-            "errors": self.__set_errors_json(self.__assessment_total_error_count, self.__assessment_errors)
-        }
+        # Compose substatus message
+        errors = self.__set_errors_json(self.__assessment_total_error_count, self.__assessment_errors)
+
+        substatus_message = self.__compose_assessment_substatus_msg(
+            activity_id=self.execution_config.activity_id, reboot_pending=self.is_reboot_pending, crit_patch_count=critsec_patch_count,
+            other_patch_count=other_patch_count, packages=assessment_packages_json, start_time=self.execution_config.start_time,
+            last_modified_time=self.env_layer.datetime.timestamp(), started_by=started_by, errors=errors)
 
         if self.vm_cloud_type == Constants.VMCloudType.ARC:
             substatus_message["patchAssessmentStatus"] = code
             substatus_message["patchAssessmentStatusString"] = status
+
```

Comment:
Marking all changes not related to this PR goal. See my other comment on what can be done with these

Reply:
this kind of change is so small, it should be overlook. lets focus on the code logic and functionalities

Reply:
Ignore this for now to look at the bigger picture. Rajasi's guidance generally applies but I'm resolving to reduce noise.